src/index.js

```diff
@@ -306,11 +310,15 @@ export default function nodeResolve ( options = {} ) {
 						browserMapCache.set(resolved, packageBrowserField);
 					}
 
+					if ( hasPackageEntry && !preserveSymlinks && resolved ) {
+						return existsAsync( resolved )
+							.then(exists => exists ? realpathAsync( resolved ) : resolved);
+					} else {
```

Comment:
No reason for else after return, but this is mostly a style question

Reply:
fixed